fabfile/setup.py

```diff
@@ -39,13 +39,16 @@ def create_ansible_env():
     Creates the ansible environment
     :return: Void
     """
+    #TODO: `"mkvirtualenv"` and `"workon"` don't work in `local` call on some platforms
     with bash():
         with settings(warn_only=True):
             local('mkvirtualenv ansible')
     with ansible():
         with settings(warn_only=True):
-            local('pip install ansible==1.9.4')
-            local('sudo ansible-galaxy install -r $FAB_PATH/../orchestration/roles/roles.txt')
+            local('pip install ansible')
+        with settings(warn_only=True):
+            local('sudo ansible-galaxy remove nodesource.node')
```

Comment:
`ansible-galaxy` doesn't have a set way of updating packages (which caused nodesource to break whenever they updated the node version). To truly do an update, we'd need to remove and then re-install roles, which is what this segment of the code does.